interface/ceed-operator.c

```diff
@@ -1493,8 +1493,8 @@ int CeedOperatorApply(CeedOperator op, CeedVector in, CeedVector out, CeedReques
         }
       }
       // Apply
-      for (CeedInt i = 0; i < op->num_suboperators; i++) {
-        CeedCall(CeedOperatorApplyAdd(op->sub_operators[i], in, out, request));
```

Comment:
side note - this probably could have been stripped out as a separate fix. I like merging fast fix PRs like that

Reply:
Yeah whoops I'm sorry about this one: See https://github.com/CEED/libCEED/pull/1313. Thanks!